samples/excel/worksheet/activeworksheet.yaml

```diff
@@ -0,0 +1,194 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Active worksheet
+description: Explore the APIs to get and set the active worksheet
+script:
+  content: |-
+    $("#setup").click(setup);
+
+    $("#get-active-worksheet").click(getActiveWorksheet);
+    $("#set-active-worksheet").click(setActiveWorksheet);
+
+    async function getActiveWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                sheet.load("name");
+                
+                await context.sync();
+
+                console.log(`The active worksheet is "${sheet.name}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function setActiveWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                sheet.load("name");
+                
+                let sheets = context.workbook.worksheets;
+                sheets.load("items");
+
+                await context.sync();
+
+                let newSheet = sheet;
```

Comment:
There is a chance that there is only 1 sheet in the worksheet and in that case newSheet will be the active worksheet.